### DISCUSSION TITLE
faq/deploy

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#21] [Announcements]
# faq/deploy

如何部署到 CDN 在编排文件 docker-compose.yaml 中设置 vanblog 容器的 VANBLOGCDN_URL 这个环境变量后，按部就班增加 CDN 即可。 image 原则上 CDN 只缓存 /_next/static 这个目录就够了。 如何安装 docker ? 可以用这个一键安装脚本: 如何在外部访问数据库 默认的数据库是不...

https://vanblog.mereith.com/faq/deploy.html

**Comment by XXJNOTFOUND** [comment]
作者大大，我看到您github里有一个解决宝塔反代的方法，在[这里](https://github.com/Mereithhh/vanblog/issues/469)
